Exception/GoogleVideoUrlException.php

```diff
@@ -15,6 +15,8 @@
  * Exception used when limit is reached on adding video
  *
  * @author David Epely <depely@prestaconcept.net>
+ *
+ * @deprecated Use \Presta\SitemapBundle\Exception\GoogleVideoException instead.
  */
 class GoogleVideoUrlException extends Exception
```

Comment:
A `@trigger_error('', E_USER_DEPRECATED);` should be added in `__construct`

Reply:
If we keep the inheritance above (`GoogleVideoException` extends `GoogleVideoUrlException`), we won't be able to trigger a deprecation in the constructor (since the depreciation would be triggered when the new exception is thrown). I think we can't preserve try/catch compatibility AND trigger a deprecation. Let me know if you prefer to not have a BC layer for exception (or to keep old exceptions).

Reply:
As I said above, I didn't see it like that, but your idea seems good to me.